src/main/java/com/example/model/controllers/TelevisionController.java

```diff
@@ -0,0 +1,79 @@
+package com.example.model.controllers;
+
+
+
+import com.example.model.models.Television;
+import com.example.model.services.TelevisionService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/televisions")
+public class TelevisionController {
+
+    private final TelevisionService televisionService;
+
+    public TelevisionController(TelevisionService televisionService){
+        this.televisionService = televisionService;
+    }
+
+
+
+    @GetMapping
+    public ResponseEntity<List<Television>> getAllTelevisions(){
+        return ResponseEntity.ok(televisionService.getAllTelevisions());
+    }
+
+    @PostMapping
+    public ResponseEntity <Television> saveTelevision(@RequestBody Television television){
+        return ResponseEntity.status(HttpStatus.CREATED).body(televisionService.saveTelevision(television));
+    }
+
+
+    @DeleteMapping("/{id}")
+    public ResponseEntity<Object> deleteTelevision(@PathVariable Long id){
```

Comment:
Het gbruik van `<Object>` is hier niet fout, maar we gebruiken liever `<?>` of `<Void>`